docs/literate/src/files/getting_started.jl

```diff
@@ -0,0 +1,288 @@
+# the compressible Euler equations in 2D for an ideal gas ([`CompressibleEulerEquations2D`](@ref))
+# with a weak blast wave as the initial condition.
+
+# Invoke Julia in a terminal.
+# And execute following code.
+
+using Trixi, OrdinaryDiffEq
+trixi_include(@__MODULE__,joinpath(examples_dir(), "tree_2d_dgsem", "elixir_euler_ec.jl"))
+
+# To observe the result of the computation, we need to use the `Plots` package and the function 
+# `plot()`, that builds a graphical representation of the solution. `sol` is a variable defined in
+# executed example and it contains a solution at a final moment of time.
+
+using Plots
+plot(sol)
+
+# To obtain list of all Trixi.jl elixirs execute
+# [`get_examples`](@ref). It returns paths to all examples.
+
+get_examples()
+
+# Editing Trixi.jl examples is the best way to start your first own investigation using Trixi.jl.
+
+# ### Files downloading
+
+# To edit example files you have to download them. Let's have a look how to download
+# `elixir_euler_ec.jl` used in the previous section from the
+# [Trixi.jl github](https://github.com/trixi-framework/Trixi.jl).
+
+# - All examples are located inside
+#   the [`examples`](https://github.com/trixi-framework/Trixi.jl/tree/main/examples) folder.
+# - Navigate to the
+#   file [`elixir_euler_ec.jl`](https://github.com/trixi-framework/Trixi.jl/blob/main/examples/tree_2d_dgsem/elixir_euler_ec.jl).
+# - Click the `Raw` button on the right side of the webpage.
+# - Right-click on any place of the newly opened webpage and choose `Save as`.
+# - Choose a folder and erase `.txt` from the file name. Save the file.
+
+# ### Files editing
+
+# For example, we will change the initial condition for calculations that occur in
+# `elixir_euler_ec.jl`. In this example we consider the compressible Euler equations:
+# ```math
+# \frac{\partial}{\partial t}
+# \begin{pmatrix}
+# \rho \\ \rho v_1 \\ \rho v_2 \\ \rho e
+# \end{pmatrix}
+# +
+# \frac{\partial}{\partial x}
+# \begin{pmatrix}
+# \rho v_1 \\ \rho v_1^2 + p \\ \rho v_1 v_2 \\ (\rho e +p) v_1
+# \end{pmatrix}
+# +
+# \frac{\partial}{\partial y}
+# \begin{pmatrix}
+# \rho v_2 \\ \rho v_1 v_2 \\ \rho v_2^2 + p \\ (\rho e +p) v_2
+# \end{pmatrix}
+# =
+# \begin{pmatrix}
+# 0 \\ 0 \\ 0 \\ 0
+# \end{pmatrix}
+# ```
+# for an ideal gas with ratio of specific heats ``\gamma`` 
+# in two space dimensions.
+# Here, ``\rho`` is the density, ``v_1``, ``v_2`` the velocities, ``e`` the specific total
+# energy, and
+# ```math
+# p = (\gamma - 1) \left( \rho e - \frac{1}{2} \rho (v_1^2+v_2^2) \right)
+# ```
+# the pressure.
+# Initial condition consists of initial values for ``\rho``, ``\rho v_1``,
+# ``\rho v_2`` and ``\rho e``.
+# One of the common initial condition for compressible Euler equations is density wave.
+# Let's implement it.
+
+# - Open the downloaded file with notepad or any other text editor.
+# - And go to the 9th line with following code:
+#   ````
+#   initial_condition = initial_condition_weak_blast_wave
+#   ````
```

Comment:
~~~suggestion
#   ```julia
#   initial_condition = initial_condition_weak_blast_wave
#   ```
~~~
This should apply Julia formatting to the source code. Please also do this for all other Julia source code snippets that you include in markdown